src/prisma/_httpx_sync_http.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+
+import httpx
+
+from ._types import Method
+from .http_abstract import AbstractResponse, AbstractHTTP
+
+
+__all__ = ('HTTP', 'Response', 'client')
+
+
+class HTTP(AbstractHTTP[httpx.Client, httpx.Response]):
+    # pylint: disable=invalid-overridden-method,attribute-defined-outside-init
+
+    session: httpx.Client
+
+    def download(self, url: str, dest: str) -> None:
+        with self.session.stream("GET", url, timeout=None) as resp:
+            with open(dest, 'wb') as fd:
+                for chunk in resp.iter_bytes():
+                    fd.write(chunk)
+
+    def request(self, method: Method, url: str, **kwargs: Any) -> 'Response':
+        return Response(self.session.request(method, url, **kwargs))
```

Comment:
Replace `data` with `content` due to deprecation warnings since httpx [v0.18.0](https://github.com/encode/httpx/blob/master/CHANGELOG.md#0180-27th-april-2021)
```suggestion
        content = kwargs.pop('data', None)
        if content is not None:
            kwargs['content'] = content

        return Response(self.session.request(method, url, **kwargs))
```